src/components/combat/Combat.js

```diff
@@ -0,0 +1,53 @@
+import combatStyle from './Combat.module.scss'
```

Comment:
It should only be called `stl` or `style` if we're importing only one stylesheet (in the component).

```suggestion
import stl from './Combat.module.scss'
```